src/java.base/share/classes/sun/util/calendar/BaseCalendar.java

```diff
@@ -277,6 +277,10 @@ void normalizeMonth(CalendarDate date) {
             long xm = 1L - month;
             year -= (int)((xm / 12) + 1);
             month = 13 - (xm % 12);
+            if (month == 13) {
+                year ++;
```

Comment:
Not that it matters, just happen to see it ;-). Is it more common to have no space before the increment operator? Code convention seems to suggest no space before increment and their operands. 
No update needed should you decide to remove it.

Reply:
Thanks. Indeed it was in the coding convention. Removed.